### ISSUE TITLE
Implement LIME for Pandas

### ISSUE THREAD

**Issue by thomasfrederikhoeck** [andersbogsnes/ml_tooling#198]
#### Problem description

It would very usefull to have LIME build in to the ml_tooling for explaining predictions. One of the current packages ELI5 doens't support pandas DataFrames.

https://eli5.readthedocs.io/en/latest/blackbox/lime.html

Checklist
---------
- [ ] I have updated ml-tooling to the newest version `pip install ml-tooling --upgrade`
- [ ] I have checked that this issue does not already exist

**Comment by andersbogsnes** [comment]
We should decide on LIME/SHAP or both

**Comment by stephanobryan** [comment]
Both would be great, only LIME would be fine, too ;-).
